Package.swift

```diff
@@ -21,12 +24,6 @@ let package = Package(
                 .product(name: "Fluent{{fluent.db.module}}Driver", package: "fluent-{{fluent.db.url}}-driver"),{{/fluent}}{{#leaf}}
                 .product(name: "Leaf", package: "leaf"),{{/leaf}}
                 .product(name: "Vapor", package: "vapor")
-            ],
-            swiftSettings: [
```

Comment:
Why are we removing this?

Reply:
@0xTim Per https://www.swift.org/server/guides/building.html#:~:text=cross-module-optimization, "it may sometimes cause performance regressions", and I can't think of any instances I could cite where I saw it have much of _any_ impact. (Also, I believe it's on by default in 5.9, possibly sooner.)

Reply:
I'm conflicted on this - on the one hand performance regressions (for release builds only), on the other hand - no data to prove it either way and it's the current 'recommended' way as per the swift.org guides. Happy to let it slide though